modules/microshift-creating-ostree-iso.adoc

```diff
@@ -19,10 +19,11 @@ Use the following procedure to create the ISO. The {op-system-ostree} Installer
 
 . Start an `ostree` container image build by running the following command:
 +
-[source,terminal,subs="attributes+"]
+[source,terminal,subs="+quotes"]
 ----
-$ BUILDID=$(sudo composer-cli compose start-ostree --ref "rhel/{op-system-version-major}/$(uname -m)/edge" minimal-microshift edge-container | awk '{print $2}')
+$ BUILDID=$(sudo composer-cli compose start-ostree --ref "rhel/{op-system-version-major}/$(uname -m)/edge" __<microshift_blueprint>__ edge-container | awk '{print $2}')<1>
 ----
+<1> Replace _<microshift_blueprint>_ with the name of your blueprint.
```

Comment:
Per the "Important" note in this section of the [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values), we shouldn't italicize user replaced values. Apparently this is an exception to the SSG, so apologies for the inconvenience if you were following that as a model:
```suggestion
$ BUILDID=$(sudo composer-cli compose start-ostree --ref "rhel/{op-system-version-major}/$(uname -m)/edge" <microshift_blueprint> edge-container | awk '{print $2}')<1>
----
<1> Replace `<microshift_blueprint>` with the name of your blueprint.
```

Reply:
Hi Sebastian,
In line 26 , using back ticks for user values is not followed.
in line 24, I do agree with the suggestion made but I would retain the changes as is to maintain consistency in the doc. I can take this suggestion in another jira and make the suggested changes to all the places in the doc and remove the italics in command.

For now I think this change is out of scope for this PR.

Also, I have made edits to line 39, 42, 44. Please review before merging this PR.

Reply:
@DanielFroehlich and @dhensel-rh Please review the lines 39,42 and 44. Changes are done for the blueprint name.

Reply:
Hi Ashwini, unless I'm mistaken, there is no place in our docs where we use italics for user-replaced values, or if there is, they incorrect and should be fixed. This is the OCP docs guidelines section on user-replaced values in command, which has an Important admonition explicitly stating to not use italics around these values: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values

Any rule in that guideline takes priority over the SSG, since that guideline is more specific to our product docs. The italics must be removed in both lines 24 and 26.

Additionally, please take a look at this section of our guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

In that table, you'll find a row for "System or software variable to be replaced by the user", which refers to user replaced values, and you'll see that they are wrapped in backticks. This means that whenever user-replaced values are used outside of command syntax, like the callout label on line 26 here, they must have backticks.

Please let me know when you've implemented the feedback in my original suggestion.

Reply:
As discussed, OCP has made the exception to this SSG rule because it was too much effort to update that repo with the italics. Microshift is a new docset and follows the SSG. So, this guideline can be ignored and the italics can be retained to maintain consistency.
cc @ShaunaDiaz

Reply:
There are two issues here: the italics in the command block (which based on discussion above I understand is a difference between OCP and MicroShift) and the backticks around the value in the callout explanation. 

The latter item seems to also be contrary to [the SSG](https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values):
"If the user-replaced value is referencing a value in code or in a command that is normally monospace, also use monospace for the user-replaced value"

If I am reading right, it sounds like this is consistently not used in the existing docs.

So, if i am following, what makes sense to do here is:
1. leave the italics alone, MicroShift uses them and that's fine
2. the omitted backticks can be tackled across the docs as a separate effort.

Is that right? Just making sure I have understood - fwiw if that is what you are proposing, I think that sounds reasonable :)

Reply:
It does.

Reply:
Yes that is right @jeana-redhat

Reply:
Thank you for confirming all that! SGTM.